packages/fork-choice/test/unit/forkChoice/getProposerHead.test.ts

```diff
@@ -236,7 +236,7 @@ describe("Forkchoice / GetProposerHead", function () {
 
       const forkChoice = new ForkChoice(config, fcStore, protoArr, {
         proposerBoostEnabled: true,
-        // proposerBoostReorgEnabled: true,
+        proposerBoostReorgEnabled: true,
```

Comment:
fix this

Reply:
> fix this

Didn't quite catch that. Can you elaborate what's wrong?

Reply:
unrelated comment: do we wanna align the CLI flag with other flags by removing the -`Enabled` suffix? could alias it for existing proposer boost flag

Reply:
nvm maybe I misread the code

Reply:
> unrelated comment: do we wanna align the CLI flag with other flags by removing the -`Enabled` suffix? could alias it for existing proposer boost flag

@nflaig Yea let's do it in another PR